devel/folly/Portfile

```diff
@@ -50,6 +50,9 @@ platform darwin {
         configure.args-append     -DCOMPILER_HAS_F_ALIGNED_NEW=OFF
         configure.cxxflags-append -fno-aligned-allocation
     }
+    if {${os.major} < 15} {
```

Comment:
I know its not strictly part of this PR, but could you move the comment

```
# https://github.com/facebook/folly/issues/864
```

to just before the ` darwin <= 16` block, and also add another comment for the new one here just briefly explaining why it is there.